frontend/model/notifications/periodicNotifications/index.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+import sbp from '@sbp/sbp'
+import { Vue } from '@common/common.js'
+// $FlowFixMe
+import { objectOf, string, isFunction } from '@model/contracts/misc/flowTyper.js'
+import { MINS_MILLIS } from '@model/contracts/shared/time.js'
+import { PERIODIC_NOTIFICATION_TYPE } from './constants.js'
+
+const { MIN1, MIN5, MIN30 } = PERIODIC_NOTIFICATION_TYPE
+
+const every1MinTimeout = { queue: [], state: {}, delay: 0.2 * MINS_MILLIS }
+const every5MinTimeout = { queue: [], state: {}, delay: 5 * MINS_MILLIS }
+const every30MinTimeout = { queue: [], state: {}, delay: 30 * MINS_MILLIS }
```

Comment:
`state: {}` object here is for the case where we want to keep some data between different consecutive timeout calls. For example, it is used for implementing `NEW_DISTRIBUTION_PERIOD` notification, where we need to store and read the `distributionDate` value from the previous time call and compare it to the value in the current call. (`this.distPeriodRecords` there). hope it sounds good.

Reply:
Oops, `0.2 * MINS_MILLIS` here is supposed to be `MILS_MILLIS` here. forgot to revert it back after testing. I will make the fix along with your upcomming change requests.